src/icalendar/parser_tools.py

```diff
@@ -2,6 +2,23 @@
 DEFAULT_ENCODING = 'utf-8'
 
 
+def from_unicode(value, encoding='utf-8') -> bytes:
```

Comment:
This seems to be a useful method. What are the inputs and what is the data it works on?

Reply:
It will accept any data type input but if that input is a string, it will encode it to bytes.